src/app/tests/TestCommandInteraction.cpp

```diff
@@ -294,6 +315,24 @@ void TestCommandInteraction::TestCommandHandlerWithProcessReceivedMsg(nlTestSuit
     commandHandler.Shutdown();
 }
 
+void TestCommandInteraction::TestCommandSenderWithProcessReceivedNotExistCommand(nlTestSuite * apSuite, void * apContext)
```

Comment:
This is not testing CommandSender at all, so is this the right method name?

Reply:
Sorry, it should be TestCommandHandler